src/components/RewardWrapper/index.tsx

```diff
@@ -446,6 +449,8 @@ export function RewardWrapper({
     networkName = chainMap[reward.chain_id.toString()].chain?.name ?? ''
   }
 
+  const canClaim = claimEnabledForQuestType && canClaimReward
```

Comment:
I think we forgot previously to add the `isRewardTypeClaimable` here as well

```suggestion
  const canClaim = claimEnabledForQuestType && canClaimReward && isRewardTypeClaimable
```